database/migrations/2021_01_01_100000_create_leads_table.php

```diff
@@ -13,16 +13,16 @@ public function up()
             $table->id();
             $table->string('lead_number')->index()->unique(); // Generated by system. This is identifier used to communicate about the quote request & for additional form pages.
             $table->string('email');
-            $table->foreignIdFor(LeadBusiness::class);
+            $table->foreignIdFor(LeadBusiness::class); // moving to other table to reverse the relationship
```

Comment:
Per our discussion, this also needs to move so Lead Business (#6) is not required for every Lead (#5) and it will allow us to have many Businesses for one Lead in the future if we want.